bin/deploy

```diff
@@ -112,10 +110,23 @@ test -f "$LSSTSW/miniconda/.packages.deployed" || ( # conda packages
     fi
 
     if [[ $BLEED_DEPLOY == true ]]; then
-        # The conda Intel MKL linked packages are intentionally avoided.
-        # See: https://jira.lsstcorp.org/browse/DM-5105
-        CONDA_BLEED_PACKAGES=(nomkl numpy scipy matplotlib requests cython sqlalchemy astropy pandas future)
+        # lsst_distrib / lsst_sims deps
+        CONDA_BLEED_PACKAGES=(numpy scipy matplotlib requests cython sqlalchemy astropy pandas future)
         ARGS+=("${CONDA_BLEED_PACKAGES[@]}")
+
+        # matplotlib pulls in qt and there are concerns about qt5 causing
+        # breakage, so we are pinning to 4.x
+        ARGS+=("pyqt==4.11.4")
```

Comment:
I disagree with this. I think we should definitely pin `pyqt` at v4 when we request a version pinned build but if I'm doing a bleeding edge build I want to know what breaks with the bleeding edge. I don't want to get confused by pinning regardless.